app/src/main/res/values/strings.xml

```diff
@@ -1,234 +1,489 @@
          performed. It indicates that the search query is running and they should wait for the result. -->
     <string name="voice_search_decoding">Searching…</string>
-    <!-- This string is displayed in a dialog box when the phrase the user spoke for the voice search
-         was not understood by the system. -->
+
+    <!-- This string is displayed in a dialog box show to the user while the Voice Search is being
+         performed. It indicates that the search query is running and they should wait for the result. -->
     <string name="voice_search_error">Sorry! I could not understand.</string>
-    <!-- This string is displayed in a dialog box after a failed voice search and is prompting the
-         user to try the voice search again. -->
+
+    <!-- This string is displayed in a dialog box after a failed Voice Search and the system is prompting the
+         user to try the Voice Search again. -->
     <string name="voice_search_try_again">Please try again.</string>
+
     <!-- This string is displayed in a dialog after the user has declined permission to use the
-         devices microphone. It explains why voice search is not working. -->
-    <string name="voice_search_permission_after_decline">Please allow microphone access to use the voice search.</string>
+         device's microphone. It explains why Voice Search is not working. -->
+    <string name="voice_search_permission_after_decline">Please allow microphone access to use the Voice Search.</string>
+
     <!-- This string is displayed in the URL bar when it is empty. It informs the user that they
          may use the URL bar to search the Web or enter a Web address. -->
     <string name="search_placeholder">Search the Web or enter address</string>
-    <!-- This string labels a button used to cancel an action. -->
+
+    <!-- This string labels a button that is used to cancel an action.
+         This appears in such contexts as dialog boxes, `<select>` menus, etc. -->
     <string name="cancel_button">Cancel</string>
-    <!-- This string labels a button that are used to approve an action. -->
+
+    <!-- This string labels a button that is used to approve an action.
+         This appears in such contexts as dialog boxes, `<select>` menus, etc. -->
     <string name="ok_button">OK</string>
-    <!-- This string labels a button to go back to the previous menu or exits the current mode -->
+
+    <!-- This string labels a button that is used to approve an action. -->
     <string name="back_button">Back</string>
+
     <!-- This string is displayed in a dialog after an application crash. -->
     <string name="crash_dialog_heading">We had an issue and crashed</string>
+
     <!-- This string explains why Mozilla would like the user to allow the crash data to be uploaded.
-         '%1$s' will be replaced at runtime with the app name. -->
+         '%1$s' will be replaced at runtime with the website's domain name. -->
```

Comment:
Same comment as Pike before: shouldn't this also be the app name?

Reply:
Fixed